tests/test_dnssec.py

```diff
@@ -927,16 +928,33 @@ def testInvalidDigestLengthCDS0(self):  # type: () -> None
                 self.assertEqual(msg, str(cm.exception))
 
 
+@unittest.skipUnless(dns.dnssec._have_pyca, "Python Cryptography cannot be imported")
 class DNSSECKeyTestCase(unittest.TestCase):
-
     def testKeyToDNSKEY(self):  # type: () -> None
         for tk in test_dnskeys:
             print(tk.command)
             key = load_pem_private_key(tk.private_pem.encode(), password=None)
             rdata1 = str(dns.dnssec.key_to_dnskey(key.public_key(), tk.algorithm))
-            rdata2 = str(dns.rdata.from_text(dns.rdataclass.IN, dns.rdatatype.DNSKEY, tk.dnskey))
-            self.assertEqual(rdata1,rdata2)
+            rdata2 = str(
+                dns.rdata.from_text(dns.rdataclass.IN, dns.rdatatype.DNSKEY, tk.dnskey)
+            )
+            self.assertEqual(rdata1, rdata2)
 
+    def testKeyRSALargeExponent(self):  # type: () -> None
```

Comment:
I suggest to use a loop in this function to improve readability

Reply:
@nils-wisiol Is 3949ae47cc10ebe7d2e98ddbb4f3125c88bfd257 easier to read?

Reply:
Excellent! :+1: